app/src/main/res/values-hi/strings.xml

```diff
@@ -328,4 +329,6 @@
     <string name="placeholder_total_rating">प्लेसहोल्डर कुल रेटिंग</string>
     <string name="invalid_credentials">अवैध प्रत्यय पत्र</string>
     <string name="invalid_credentials_title">अमान्य क्रेडेंशियल्स शीर्षक</string>
+
```

Comment:
Unneeded change, remove these lines

Reply:
You mean, I need to remove those 3 lines of string?

Reply:
The blank lines

Reply:
Ok, I will remove those blank lines and commit soon.